front/app/components/Form/Components/Layouts/CLPageLayout.tsx

```diff
@@ -154,12 +154,12 @@ const CLPageLayout = memo(
               currentStep === index && (
                 <StyledFormSection key={index}>
                   {page.options.title && (
-                    <Title variant="h2" mt="0" mb="24px" color="tenantText">
+                    <Title variant="h2" mt="0" mb="24px" color="tenantPrimary">
                       {page.options.title}
                     </Title>
                   )}
                   {page.options.description && (
-                    <Box mb="48px">
+                    <Box mb={page.elements.length >= 1 ? '48px' : '28px'}>
```

Comment:
Removes a big white space at the bottom of a survey page when there are no questions on the page.